atc/db/resource_check_rate_limiter.go

```diff
@@ -88,7 +89,8 @@ func (limiter *ResourceCheckRateLimiter) Limit() rate.Limit {
 func (limiter *ResourceCheckRateLimiter) refreshCheckLimiter() error {
 	var count int
 	err := psql.Select("COUNT(id)").
-		From("resource_config_scopes").
+		From("resources").
+		Where(sq.Eq{"active": true}).
```

Comment:
hmm the tests don't cover this clause - maybe it could be extended to demonstrate that the rate changes when fewer resources are configured? should be easy enough with another function that just reduces `checkableCount` and saves a new (smaller) set of resources with dbtest.

Reply:
Fixed by resetting the pipeline with an empty resource list, so that all existing resources will be set to inactive.